tests/test_cli.py

```diff
@@ -78,3 +79,26 @@ def test_startproject_invalid_project_name():
         result = runner.invoke(cli, ['startproject', project_name])
 
         assert result.exit_code == 2
+
+
+def test_version_option():
+    runner = CliRunner()
+    result = runner.invoke(cli, ['--version'])
+
+    assert bottery.__version__ in result.output
+    assert result.exit_code == 0
+
+
+def test_no_options_shows_help_message():
+    """
+    Test if CLI shows the help message when no option or command is
+    given.
+    """
+    runner = CliRunner()
+    result = runner.invoke(cli)
+    ctx = click.Context(cli, info_name='cli')
+
+    # Couldn't use == to compare command output with helper message
+    # because of a \n at the end :/
```

Comment:
In those cases where I don't care about a newline or whitespace, I just `strip()` one of the sides.